sdk/synapse/azure-synapse-artifacts/azure/synapse/artifacts/aio/_artifacts_client.py

```diff
@@ -31,6 +31,7 @@
 from .operations import SqlPoolsOperations
 from .operations import BigDataPoolsOperations
 from .operations import IntegrationRuntimesOperations
+from .operations import LibraryOperations
```

Comment:
Is it possible to exclude library operations in this release? I remember according to discussion, we will not add new features in this release. @lmazuel , am I right?